tutor/controllers/forms.py

```diff
@@ -32,5 +32,6 @@ class AddCourse(FlaskForm):
 
 class resourceForm(FlaskForm):
     title = StringField('Title', validators=[DataRequired()])
-    content = TextAreaField('Content', validators=[DataRequired()])
+    content = TextAreaField('Description', validators=[DataRequired()])
+    link = TextAreaField('Link', validators=[DataRequired()])
```

Comment:
If link needs to be a working HTTP URL (An I think it is because you're putting it in a "href=" attribute) - we probably need validation for that

Reply:
Changed it to display the title